public/claimlist.php

```diff
@@ -0,0 +1,412 @@
+$sortClaimDate = ($sortType == 8) ? 18 : 8;      // Claim Date
+$sortExpirationDate = ($sortType == 9) ? 19 : 9; // Expiration Date
+
+if (!empty($gameID)) {
+    getGameTitleFromID($gameID, $gameTitle, $consoleID, $consoleName, $forumTopic, $gameData);
+}
+
+sanitize_outputs(
+    $gameTitle,
+    $consoleName,
+);
+
+RenderHtmlStart();
+if ($type == 0) {
+    RenderHtmlHead("Claim List");
+} else {
+    RenderHtmlHead("Expiring Claims");
+}
+?>
+<body>
+<?php
+RenderHeader($userDetails);
+?>
+<div id='mainpage'>
+    <div id='fullcontainer'>
+        <?php
+        RenderErrorCodeWarning($errorCode);
+        if ($type == 0) { // Show standard claim list
+            echo "<h3>Claim List - $activeClaimCount Active Claims</h3>";
+            echo "<h4>Filters - $totalClaims Claims Filtered</h4>";
+
+            // Claim Type filter
+            echo "<div class='embedded mb-1'>";
+            echo "<div>";
+            echo "<b>Claim Type:</b> ";
+            if ($primaryClaim) {
+                echo "<b><a href='/claimlist.php?s=$sortType&f=" . ($claimFilter & ~(1 << 0)) . "&u=$username&g=$gameID'>*Primary</a></b> | ";
+            } else {
+                echo "<a href='/claimlist.php?s=$sortType&f=" . ($claimFilter | (1 << 0)) . "&u=$username&g=$gameID'>Primary</a> | ";
+            }
+
+            if ($collaborationClaim) {
+                echo "<b><a href='/claimlist.php?s=$sortType&f=" . ($claimFilter & ~(1 << 1)) . "&u=$username&g=$gameID'>*Collaboration</a></b>";
+            } else {
+                echo "<a href='/claimlist.php?s=$sortType&f=" . ($claimFilter | (1 << 1)) . "&u=$username&g=$gameID'>Collaboration</a>";
+            }
+            echo "</div>";
+
+            // Set Type filter
+            echo "<div>";
+            echo "<b>Set Type:</b> ";
+            if ($newSetClaim) {
+                echo "<b><a href='/claimlist.php?s=$sortType&f=" . ($claimFilter & ~(1 << 2)) . "&u=$username&g=$gameID'>*New Set</a></b> | ";
+            } else {
+                echo "<a href='/claimlist.php?s=$sortType&f=" . ($claimFilter | (1 << 2)) . "&u=$username&g=$gameID'>New Set</a> | ";
+            }
+
+            if ($revisionClaim) {
+                echo "<b><a href='/claimlist.php?s=$sortType&f=" . ($claimFilter & ~(1 << 3)) . "&u=$username&g=$gameID'>*Revision</a></b>";
+            } else {
+                echo "<a href='/claimlist.php?s=$sortType&f=" . ($claimFilter | (1 << 3)) . "&u=$username&g=$gameID'>Revision</a>";
+            }
+            echo "</div>";
+
+            // Claim Status filter
+            echo "<div>";
+            echo "<b>Claim Status:</b> ";
+            if ($activeClaim) {
+                echo "<b><a href='/claimlist.php?s=$sortType&f=" . ($claimFilter & ~(1 << 4)) . "&u=$username&g=$gameID'>*Active</a></b> | ";
+            } else {
+                echo "<a href='/claimlist.php?s=$sortType&f=" . ($claimFilter | (1 << 4)) . "&u=$username&g=$gameID'>Active</a> | ";
+            }
```

Comment:
You might be interested in the changes I made in #970 to make this a little more manageable.

Reply:
Updated to follow ticket manager updates as suggested.